### PR TITLE
Feat 690 toggle filtering

### PR DESCRIPTION
**Pull request by chriswk**
Need to find a way to unit test this and not just e2e;  think the answer is to move the featureTagStore methods into the featureStore again. That way our fake stores would also have the connection between features and tags.

The filter query isn't too bad. the service validates the parameters and keeps only parameters we know/allow to query on.

### CODE REVIEW

File: lib/routes/admin-api/feature.js [hunk 1]
```diff
@@ -9,6 +9,18 @@ const {
...
+    'description',
+    'type',
+    'project',
+    'enabled',
+    'stale',
+    'strategies',
+    'variants',
+    'created_at',
+    'last_seen_at',
+];
```
Comment (ivarconr) [inline lib/routes/admin-api/feature.js:23]: I feel this is leaking the column names from the stores to the controller. Could we just define the fields as they are returned from the stores/serivices and have the store perform the column mapping themselves (from camelCase to snake_case).

**Comment by ivarconr** [comment]
> Need to find a way to unit test this and not just e2e; think the answer is to move the featureTagStore methods into the featureStore again. That way our fake stores would also have the connection between features and tags.

Yes we need a way to unit test. One could argue that unit-testing a controller should use service-mocks. But we still need to test the services and then we still require store-mocks. 

(as discussed off-line: In the future it would be good to be able to offer "in-memory" stores (for local dev). Then we could utilize the in-memory implementation for unit tests.)

**Review by ivarconr — APPROVED** [review]

**Review by ivarconr — APPROVED** [review]

File: lib/routes/admin-api/feature.js (hunk 1 above)
Comment (chriswk) [reply lib/routes/admin-api/feature.js:23]: good point; the controller shouldn't actually know the column names. I'm open to suggestions as to how to do this.

Comment (ivarconr) [reply lib/routes/admin-api/feature.js:23]: ca6e07a82b69f65bb4fbfa2d45e3101acef3da8a seems like a good solution.

**Pull request merged**